spec/features/admin_creates_other_admins_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'rails_helper'
+
+# [ X] As an authenticated Admin
+# I want to make another user an admin
+# So more people can control the data
+
+feature "Admin makes a user an admin" do
+
+  let!(:user1) { FactoryGirl.create(:user, admin: true) }
+  let!(:user2) { FactoryGirl.create(:user) }
```

Comment:
I'd like to see these objects named according to their test purpose. So `user1` becomes `admin_user`, `user2` becomes `non_admin_user_1`, etc. That way the tests read really smoothly without needing to refer to the `let` to understand the special attributes of each object.